doc/multiple_cursors.txt

```diff
@@ -177,30 +128,43 @@ These keys must be manually listed because vim doesn't provide a way to
 automatically see which keys _start_ mappings, and trying to run motion commands
 such as `j` as if they were operator-pending commands can break things.
 
+==============================================================================
+5. Interactions with other plugins                    *multiple-cursors-other-plugins*
+
+Other plugins may be incompatible in insert mode. That is why we provide
+hooks to disable those plug-ins when vim-multiple-cursors is active:
+
+For example, if you are using `Neocomplete`, add this to your vimrc to prevent
+conflict:
+ >
+    function! Multiple_cursors_before()
```

Comment:
I'd give more examples here that demonstrate how to deactivate other plugins. For example, I have this in my vimrc (YCM is disabled as I don't use it anymore):
```
function! Multiple_cursors_before()
    " let s:old_ycm_whitelist = g:ycm_filetype_whitelist
    " let g:ycm_filetype_whitelist = {}
    let g:deoplete#disable_auto_complete = 1

    if exists('*SwoopFreezeContext') != 0
        call SwoopFreezeContext()
    endif
endfunction
function! Multiple_cursors_after()
    " let g:ycm_filetype_whitelist = s:old_ycm_whitelist
    let g:deoplete#disable_auto_complete = 0

    if exists('*SwoopUnFreezeContext') != 0
        call SwoopUnFreezeContext()
    endif
endfunction
```
I know it's simple but as I'm not VimL pro myself, I love it when plugin authors give concrete examples that I can either directly use or slightly adapt without dwelling for hours in vim help.

Reply:
It's a README, and cannot feature all the examples for all the plugins.
However, we could maybe send a link to a Twiki that list all the plugin integration examples ?

Reply:
This may be my personal preference but I love when there is a lot examples in the README, when you have them in one place at your hand. For example, I enjoy reading `deoplete` help as Shougo puts a lot of examples there and I'm glad he does that. It helped me many times and I didn't have to leave vim and go into a browser, search the wiki, the issues, stackoverflow.
Besides, there is only a handful of plugins that require such integration. And some of them are already using autocmd so I don't expect to see many new entries here.

Reply:
Ok, i wont do it on this PR. But feel free to create one when this cleanup is done ☺